### PR TITLE
feat(connectAccountLogin): #63 Modified user model and update the code

### PR DESCRIPTION
**Pull request by VenkatVenkatesh**
- Changed the user model file

- Implemented functionality to save the connected user data to primary user collection and primary userSocial collection.

### CODE REVIEW

File: web/src/app/services/auth.service.ts [hunk 1]
```diff
@@ -90,15 +90,19 @@ export class AuthService {
       profile
     };
 
+
```
Comment (eddiejaoude) [inline web/src/app/services/auth.service.ts:93]: Extra line not needed (won't block PR though, can fix in future PR)

File: web/src/app/models/user.model.ts [hunk 2]
```diff
@@ -1,23 +1,8 @@
+import { Account } from './account.model';
 export interface User {
```
Comment (eddiejaoude) [inline web/src/app/models/user.model.ts:2]: Extra line needed (won't block PR though, can fix in future PR)

File: web/src/app/services/auth.service.ts [hunk 3]
```diff
@@ -123,13 +127,15 @@ export class AuthService {
...
 
     const normalisedUser = {
-      displayName: user.displayName,
+      twitter: {
+        displayName: user.displayName,
+        photoUrl: user.photoURL,
+        creationAt: user.metadata.creationTime,
+        lastSignInAt: user.metadata.lastSignInTime,
+        additionalUserInfo,
+        credentials,
```
Comment (eddiejaoude) [inline web/src/app/services/auth.service.ts:136]: Ending `,` not consistent. I prefer using this, but it was removed in many places in a previous PR (won't block PR though, can fix in future PR)

**Review by eddiejaoude — APPROVED** [review]

File: web/src/app/services/auth.service.ts (hunk 3 above)
Comment (VenkatVenkatesh) [reply web/src/app/services/auth.service.ts:136]: okay @eddiejaoude . Will change all through the code

**Pull request merged**